### DISCUSSION TITLE
How to run RKE2 on single node?

### DISCUSSION THREAD

**Discussion by Torkolis** [rancher/rke2#2644] [Q&A] [answered]
Is it possible to use RKE2 on a single node?
I have followed the rke2 docs and installed rke2 and started the rke2-server.service but the node runs in master mode, when I try to create a pod it stays in pending state and kubernetes says that the pod does not have a host assigned.
I have found some tips on the web saying that I should untaint the node with the master taint, but it does not work with rke2, because it says that it can't find the master taint.

**Comment by brandond** [comment, marked as answer]
A single-node cluster is able to run workloads pods by default. If you describe the pod, it should tell you why it hasn't scheduled it to a node yet.

**Reply by Torkolis** [reply]
Thanks, that answers my question. regarding pending state I found the issue (which was regarding pv&pvc in my case)
